src/components/layout/responsive-row.tsx

```diff
@@ -7,17 +7,44 @@ const PanelContainer = styled.div`
   ${({ theme }) =>
     theme.screens.sm
       ? css`
-          flex-direction: column;
+          flex-flow: column;
           align-items: start;
         `
       : css`
-          flex-direction: row;
+          flex-flow: row;
+          align-items: center;
           justify-content: center;
         `};
 `;
 
+const PanelContainerMD = styled.div`
```

Comment:
this component is unnecessary we can modify the responsive row to suit all these needs